libs/sdk-backend-bear/src/convertors/dateFormatting/defaultDateFormatter.ts

```diff
@@ -0,0 +1,21 @@
+// (C) 2020 GoodData Corporation
+import format from "date-fns/format";
+import { UnexpectedError } from "@gooddata/sdk-backend-spi";
+
+import { DateFormatter } from "./types";
+import { dateFormats, DEFAULT_DATE_FORMAT } from "./dateValueParser";
+
+/**
+ * Creates a default date formatting function.
+ * @public
+ */
+export const createDefaultDateFormatter = (dateFormat: string = DEFAULT_DATE_FORMAT): DateFormatter => {
```

Comment:
the date formatter is used alot in this feature, on many repos.
Should we move this to another location, and reuse everywhere?

Reply:
At this time, the const DEFAULT_DATE_FORMAT is being used in sdk-ui-filters and sdk-backend-bear. As both "sdk-ui-filters" and "sdk-backend-bear" depend on "sdk-model", I think we should move the definition of const DEFAULT_DATE_FORMAT to "sdk-model", what do you think?

Reply:
Totally agree

Reply:
Do we really want to tie the default date format to be the same across different background implementations? I'm not saying we don't I'm just not sure. What do you think @lupko?

Reply:
So it turns out, that moving this to sdk-model would probably be not beneficial, we do not want to make this assumption that all the backend have the same default date format.